cmd/crio/main.go

```diff
@@ -246,6 +246,11 @@ func main() {
 		if err := version.WriteVersionFile(config.VersionFilePersist); err != nil {
 			logrus.Fatal(err)
 		}
+		// Finally, we clear out the shutdown file
+		if err := os.Remove(config.CleanShutdownFile); err != nil {
```

Comment:
let's play safe here and add a:

```
	f, err := os.OpenFile(filepath.Dir(config.CleanShutdownFile), os.O_RDONLY, 0755)
	if err != nil {
		...
	}
	defer f.Close()
	
	if err = syscall.Fsync(int(f.Fd())); err != nil {
		...
	}
```
after the file is removed, so we are sure the parent directory is synced to disk